scripts/AppScriptTemplate

```diff
@@ -560,5 +560,7 @@ while [ "${EXIT_STATUS}" -eq "${RESTART_CODE}" ] ; do
     EXIT_STATUS=$?
     [ -n "${DEBUG}" ] && echo Exit Status: "${EXIT_STATUS}" | tee -a ${launcher_log}
 done
-
+if [ $EXIT_STATUS -eq 200 ] ; then
+    sudo shutdown -h 
```

Comment:
Please add time of "now" to this, as in:
`sudo shutdown -h now
`The time defaults to "+1" (1 minute) when not specified.